src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -62,10 +63,27 @@
         " dataset.id, name, description, default_profile_id, project_resource_id, application_resource_id, " +
             "created_date, ";
 
-    private static final String datasetStorageQuery = "(SELECT jsonb_agg(sr) " +
-            "FROM (SELECT region, cloud_resource as \"cloudResource\", " +
-            "lower(cloud_platform) as \"cloudPlatform\", dataset_id as \"datasetId\" " +
-            "FROM storage_resource WHERE dataset_id = dataset.id) sr) AS storage ";
+    private static final String datasetStorageQuery = "(SELECT jsonb_agg(json_build_object( " +
+        "'region', region, " +
+        "'cloudResource', cloud_resource, " +
+        "'cloudPlatform', cloud_platform, " +
+        "'datasetId', dataset_id)) " +
+        "FROM storage_resource WHERE dataset_id = dataset.id) AS storage, ";
```

Comment:
I'm confused by this change - Do you know why it might have been formatted like that to begin with?

Reply:
This is just a cleaner way of constructing the JSON object within the query, avoiding the escaped quotes and repeated `AS` keywords.

Reply:
Oh - so does the "json_build_object" call allow you to use single quotes instead of double? It looks like we're also ditching the "lower" on cloud_platform. Is that no longer needed?

Reply:
To the first point, yes! The reformatting of one entry per line is just a but more readable IMO.

To the second point, https://github.com/DataBiosphere/jade-data-repo/pull/989#discussion_r664773809